cardano-testnet/src/Testnet/Components/Query.hs

```diff
@@ -302,13 +302,13 @@ findAllUtxos
 findAllUtxos epochStateView sbe = withFrozenCallStack $ do
   AnyNewEpochState sbe' newEpochState <- getEpochState epochStateView
   Refl <- H.leftFail $ assertErasEqual sbe sbe'
-  pure $ fromLedgerUTxO $ newEpochState ^. L.nesEsL . L.esLStateL . L.lsUTxOStateL . L.utxosUtxoL
+  pure $ fromLedgerUTxO' $ newEpochState ^. L.nesEsL . L.esLStateL . L.lsUTxOStateL . L.utxosUtxoL
```

Comment:
```suggestion
  pure . unUTxO . fromLedgerUTxO $ newEpochState ^. L.nesEsL . L.esLStateL . L.lsUTxOStateL . L.utxosUtxoL
```
`fromLedgerUTxO can be removed. 

This is optional - we can do it in a follow-up PR.

Reply:
ok, I'm leaving it as-is for now.